src/test/java/uk/gov/companieshouse/service/dissolution/DissolutionServiceTest.java

```diff
@@ -48,9 +52,10 @@
     public static final String IP = "192.168.0.1";
     public static final String EMAIL = "user@mail.com";
     public static final String OFFICER_ID = "abc123";
+    public static final String ON_BEHALF_NAME = "on_behalf_name";
```

Comment:
does this file need reverted?

Reply:
Since I touched the file already, sonar will complain about "public" modifiers for test classes. So I only lefte these changes and removed the others.